handler.go

```diff
@@ -38,11 +39,21 @@ func newHandler(config []byte) (*handler, error) {
 		return nil, err
 	}
 	for _, e := range h.m {
-		if e.Display != "" {
-			continue
-		}
-		if strings.Contains(e.Repo, "github.com") {
+		switch {
+		case e.Display != "":
+			// Already filled in.
+		case strings.HasPrefix(e.Repo, "https://github.com/"):
 			e.Display = fmt.Sprintf("%v %v/tree/master{/dir} %v/blob/master{/dir}/{file}#L{line}", e.Repo, e.Repo, e.Repo)
+		case strings.HasPrefix(e.Repo, "https://bitbucket.org"):
+			e.Display = fmt.Sprintf("%v %v/src/default{/dir} %v/src/default{/dir}/{file}#{file}-{line}", e.Repo, e.Repo, e.Repo)
+		}
+		switch {
+		case e.VCS != "":
+			// Already filled in.
+		case strings.HasPrefix(e.Repo, "https://bitbucket.org/"):
+			e.VCS = "hg"
```

Comment:
Should we default to any VCS? Should we have different VCS defaults for different providers? GitHub is the only provider here that is exclusively git, so I understand if we default to git for them. But every other default looks unnecessary to me. Thoughts?